frontend/src/components/MainMenu/MainMenu.jsx

```diff
@@ -68,7 +69,8 @@ const mapStateToProps = ({ router, reducer_MainMenu }, ownProps) => ({
   location: router.location,
   mainMenu: ownProps.mainMenu,
   isExpanded: ownProps.isExpanded,
-  toggle: ownProps.toggle
+  toggle: ownProps.toggle,
+  isMainMenuExpanded: reducer_MainMenu.isMainMenuExpanded
```

Comment:
I removed this new variable as the ownProps.isExpanded is set to this exactly value when called.